shared-libs/rules-engine/test/integration.spec.js

```diff
@@ -480,6 +496,70 @@ describe('Rules Engine Integration Tests', () => {
       pass: 1,
     });
   });
+
+  it('targets on interval turnover', async () => {
```

Comment:
Can you reword this it block title? It's not saying what the test is proving, just that it's roughly about the subject of targets when the interval rolls over.

Reply:
Reworded!